roles/ucp/tasks/cleanup.yml

```diff
@@ -9,6 +9,8 @@
   with_items:
     - "{{ ucp_fingerprint_file }}"
     - "{{ ucp_instance_id_file }}"
+    - "{{ ucp_certificate_file }}"
+    - "{{ ucp_fifo_file }}"
```

Comment:
this might be a merge thing but we don't need to cleanup `ucp_fifo_file` here as it is cleaned up when UCP service is stopped.